Assignment2Conditionals/src/assignment2conditionals/Assignment2Conditionals.java

```diff
@@ -0,0 +1,107 @@
+package assignment2conditionals;
+
+import java.util.Scanner;
+
+public class Assignment2Conditionals 
+{
+
+    public static void main(String[] args) 
+    {
```

Comment:
I would like to point you to our styleguides indentation style https://github.com/WCCCEDU/twitter-commons/blob/master/src/java/com/twitter/common/styleguide.md#indent-style

This dictates we use the **"one true brace style"** meaning we keep the opening brace on the same line as the statement it describes

```java
public static void main(String[] args){
}
```